phpmyadmin/elegant-sql-fixes.md

```diff
@@ -0,0 +1,37 @@
+---
+description: >-
+  How to elegantly fix things (broken level view, post-raid, etc.) with SQL and phpMyAdmin.
+---
+So, your GDPS kinda got messed up? Don't worry, I got the page for you! This will show you elegant, simple and swift fixes to even the most tedious-to-fix issues!
+
+## Levels not appearing when you click search?
+You might know this is because of the `starCoins` or `starDemon` value being 0 nor 1. But is there a simple way to fix the levels all at once instead of manually doing it for each and every level?
+The answer might be indeed very shocking - it's yes! But how?
+Well, it's a simple snippet of SQL code, of course!
+```sql
+-- Fix starCoins
+UPDATE `levels` SET `levels`.`starCoins` = 1 WHERE `starCoins` > 1; 
+UPDATE `levels` SET `levels`.`starCoins` = 0 WHERE `starCoins` < 0;
+
+-- Fix starDemon
+UPDATE `levels` SET `levels`.`starDemon` = 1 WHERE `starDemon` > 1;
+UPDATE `levels` SET `levels`.`starDemon` = 0 WHERE `starDemon` < 0;
+```
+
+## Raided and want to remove the residue users?
+If you know the exact user ID of the first residue user (for this example, it's 40) and the last residue users (for this example, it's 12345), you can remove the users simply and elegantly.
```

Comment:
a bit scary solution because you highly assume that there are no real users in-between those accounts. probably won't matter for smaller GDPSs due to a lower frequency of new users, but I'd still be careful with that. alternatives to this such as checking by repeating names or similar IPs etc etc